### ISSUE TITLE
Duxbury

### ISSUE THREAD

**Issue by blindangel1961** [nvaccess/nvda#10226]
When you write a file in grade 2 braille in duxbury, NVDA tried to read it as text. For example: the th sign in braille is a ? You can only get it to read that by arrowing through the line. I need to be able to listen to the file that i type in Grade 2 braille and it read the braille not the characters in print. 
I would really appreciate work on this program, as I use Duxbury quite frequently and NVDA is my main source for a screen reader. 
My old laptop that still has a copy of window Eyes is slowly dying. Once it no longer works, I cannot use Duxbury unless I learn how to use JAWS. 
Thank you for all you do to develop NVDA!

**Comment by Qchristensen** [comment]
Some more information from the original reporter:

Duxbury Version 11.1
Windows 7
NVDA "Latest version" (2019.2?)

**Comment by jage9** [comment]
I'm trying to remember how this was achieved in Window-Eyes, but I believe it is essentially a different character dictionary for the translated text screen. Duxbury may be interested in writing code for this. Have you talked to anyone there about this?

**Comment by leonardder** [comment]
I'd be best if Duxbury would write unicode braille to the screen instead of characters that are probably shown using a particular font. We'd probably need a proper translation table to make this work.

**Comment by jage9** [comment]
Referencing #6341 which discusses adding the necessary symbols. While Duxbury does not currently use these, other apps do.
